content/blog/2020-09-17-delivering-learnr-tutorials-in-a-package/index.Rmarkdown

```diff
@@ -0,0 +1,322 @@
+---
+title: How to deliver learnr tutorials in a package
+author: Desirée De Leon
+date: '2020-09-17'
+slug: delivering-learnr-tutorials-in-a-package
+categories:
+  - package
+  - teach
+tags:
+  - rmarkdown
+  - teach
+  - learnr
+description: |
+  A step by step guide to create a package for a learnr tutorial.
+photo:
+  url: https://unsplash.com/photos/sxNt9g77PE0
+  author: Erda Estremera
+---
+
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(fig.align = "center", out.width = "50%")
+```
+
+## A package for the learnr occasion
```

Comment:
I don't quite understand what "learnr occasion" means, how about removing this section heading, I think it would read fine to not have it